src/components/index.js

```diff
@@ -1,5 +1,11 @@
 import NavigationTop from "./navigation-top/NavigationTop";
 import Footer from "./footer/Footer";
+import ProductCard from "./product-card/ProductCard";
+import Filters from "./filters/Filters";
+import ProductList from "./product-list/ProductList"
 
 export { NavigationTop };
 export { Footer };
+export { ProductCard };
+export { Filters };
+export { ProductList };
```

Comment:
These can be composed in single export line

Reply:
Done